heft-plugins/heft-lint-plugin/src/LintPlugin.ts

```diff
@@ -17,10 +17,19 @@ import type {
   ITypeScriptPluginAccessor
 } from '@rushstack/heft-typescript-plugin';
 
+import {
+  readConfigFile,
+  parseJsonConfigFileContent,
+  sys,
+  createProgram,
+  type ParsedCommandLine
+} from 'typescript';
```

Comment:
We forbid directly importing `typescript` in `heft-typescript-plugin` and `heft-lint-plugin` to facilitate individual projects having their own version of TypeScript. You'll need to use the same custom import logic that we use to load the project's version of eslint.

Reply:
Done in 429547a